laia/decoders/ctc_language_decoder.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict, List
+
+import numpy as np
+import torch
+from torchaudio.models.decoder import ctc_decoder
+
+from laia.losses.ctc_loss import transform_batch
+
+
+class CTCLanguageDecoder:
```

Comment:
Please add a docstring on how to use this class, and what it achieves

Reply:
Done